release/pkg/assets_capc.go

```diff
@@ -130,7 +130,6 @@ func (r *ReleaseConfig) GetCapcAssets() ([]Artifact, error) {
 func (r *ReleaseConfig) GetCloudStackBundle(imageDigests map[string]string) (anywherev1alpha1.CloudStackBundle, error) {
 	cloudstackBundleArtifacts := map[string][]Artifact{
 		"cluster-api-provider-cloudstack": r.BundleArtifactsTable["cluster-api-provider-cloudstack"],
-		"kube-rbac-proxy":                 r.BundleArtifactsTable["kube-rbac-proxy"],
```

Comment:
We have a follow up item regarding the kube rbac proxy that will add this back in?

Reply:
I don't think we have any need for kube-rbac-proxy inside the cloudstack bundle since we've moved to v1beta1. We do plan to add kube-vip however